mmv1/third_party/terraform/tests/resource_gkehub_feature_test.go.erb

```diff
@@ -0,0 +1,1092 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+  gkehub "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/gkehub/beta"
+)
+
+func TestAccGKEHubFeature_gkehubFeatureMciUpdate(t *testing.T) {
+  // VCR fails to handle batched project services
+  skipIfVcr(t)
+  t.Parallel()
+
+  context := map[string]interface{}{
+    "random_suffix":   randString(t, 10),
+    "org_id":          getTestOrgFromEnv(t),
+    "billing_account": getTestBillingAccountFromEnv(t),
+  }
+
+  vcrTest(t, resource.TestCase{
+    PreCheck:  func() { testAccPreCheck(t) },
+    Providers: testAccProvidersOiCS,
+    CheckDestroy: testAccCheckGKEHubFeatureDestroyProducer(t),
+    Steps: []resource.TestStep{
+      {
+        Config: testAccGKEHubFeature_gkehubFeatureMciUpdateStart(context),
+      },
+      {
+        ResourceName:      "google_gkehub_feature.feature",
+        ImportState:       true,
+        ImportStateVerify: true,
+      },
+      {
+        Config: testAccGKEHubFeature_gkehubFeatureMciUpdateEnd(context),
+      },
+      {
+        ResourceName:      "google_gkehub_feature.feature",
+        ImportState:       true,
+        ImportStateVerify: true,
+      },
+    },
+  })
+}
+
+func testAccGKEHubFeature_gkehubFeatureMciUpdateStart(context map[string]interface{}) string {
+  return projectSetup(context) + Nprintf(`
```

Comment:
It may be worth pulling cluster setup into a shared definition similar to `projectSetup`, that way reading the updates will be easier